source/DistanceMap.cpp

```diff
@@ -26,8 +26,17 @@ using namespace std;
 // it is a limit on how many systems should be returned. If it is below zero
 // it specifies the maximum distance away that paths should be found.
 DistanceMap::DistanceMap(const System *center, int maxCount, int maxDistance)
-	: center(center), maxCount(maxCount), maxDistance(maxDistance), useWormholes(false)
 {
+	*this = DistanceMap(center, false, false, false, maxCount, maxDistance);
+}
+
+
+
+DistanceMap::DistanceMap(const System *center, bool useWormholes, bool useJumpDrive, bool onlyUnrestrictedWormholes, int maxCount, int maxDistance)
+	: center(center), useWormholes(useWormholes), onlyUnrestrictedWormholes(onlyUnrestrictedWormholes), maxCount(maxCount), maxDistance(maxDistance)
+{
+	if(useJumpDrive)
+		jumpFuel = 200;
```

Comment:
I chose this because it's the default jump drive fuel cost, so it should proceed as though it were calculating a route for a ship with a normal jump drive.

Reply:
Sure... for vanilla. There's no actual default value, just a convention we have from the game data we provide. There's no guarantee that 200 is a reasonable value for any particular set of active game datafiles.

I'm not particularly certain there is a better option than a magic number here, but at least tack a comment on it identifying the origin of the value.

Reply:
The game actually does use default values of 100, 150, and 200 for hyperdrives, scram drives, and jump drives if no jump fuel is provided by the outfit.
https://github.com/endless-sky/endless-sky/blob/ee7d6ce2d6f0baa855b3c7cf70ec8ed289ccd0a7/source/Ship.cpp#L3149-L3171

Reply:
Added a comment explaining why that's done.